src/main/java/school/lemon/changerequest/java/introduction/hw1/Task7.java

```diff
@@ -3,6 +3,30 @@
 
 public class Task7 {
     public static String getDayTitle(int dayIndex) {
-        return "";
+        String day;
+        switch (dayIndex){
+            case 1: day = "Monday";
```

Comment:
It's OK, but you also can use simple `return "Monday";` and there will be no need in break statements and additional variable.